Dependencies/xr/Include/XR.h

```diff
@@ -146,6 +146,24 @@ namespace xr
                     bool PoseTracked{ false };
                 };
 
+                struct Button
```

Comment:
Does it make sense to have Button be a part of Frame as it is here? It could probably be embedded within the Gamepad struct, or alongside Ray and Anchor outside of System. Since only Gamepad uses it, I'd lean towards including it inside Gamepad.

Reply:
fair enough! i will move it